examples/test_odx/test_client_odx.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import time
+import typing
+import unittest
+
+import pykiso
+from pykiso.auxiliaries import uds_client_aux
+
+# helper objects to build callbacks can be imported from the pykiso lib
+from pykiso.lib.auxiliaries.udsaux import UdsCallback, UdsServerAuxiliary
+
+
+@pykiso.define_test_parameters(suite_id=1, case_id=1, aux_list=[uds_client_aux])
+class ExampleUdsServerTest(pykiso.BasicTest):
+    def setUp(self):
+        pass
+
+    def test_run(self):
+        """
```

Comment:
It's a one line docstring "Sent a request to a UDS server"""

Reply:
added better doc string